aztk/spark/models/models.py

```diff
@@ -202,30 +207,36 @@ def __init__(self, cloud_task: batch_models.CloudTask):
         self._e_tag = cloud_task.e_tag
 
 
-class JobConfiguration:
+class JobConfiguration(ConfigurationBase):
     def __init__(
             self,
-            id,
-            applications,
-            vm_size,
+            id = None,
```

Comment:
The reason these values had no default is because they are required. Why are we defaulting to `None`? To facilitate merging?

Reply:
Yeah this means we cant specify the default object, though I guess it could set it to none